pages/submitted.tsx

```diff
@@ -8,7 +8,7 @@ const Submitted: NextPage = () =>  (
             <section className={scss.content}>
                 <h1 className={scss.submitHeader}>Your Test Has Been Submitted</h1>
                 <h2 className={scss.submitMessage}>We will review your results and get back to you by email shortly</h2>
-                <h1 className={scss.submitHeader}>Thank You!</h1>
+                <h3 className={scss.thankyou}>Thank You!</h3>
```

Comment:
This implies that our page has
- A heading: "Your Test Has Been Submitted"
Then within that, there is a section titled "We will review your results and get back to you by email shortly"
And again within that, there is a subsection title "Thank You!"

I not sure that's quite what's happening though.

I think the `h1` is right, but then we don't actually have any heirachy on the rest of the page - isn't it really just 2 paragraphs of text then?